Resources/Prototypes/_Goobstation/Catalog/uplink_catalog.yml

```
@@ -638,6 +650,10 @@
     whitelist:
     - Clown
     - Mime
+  - !type:BuyerWhitelistCondition
+    blacklist:
+      components:
+      - NukieBundle
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)